tests/compile-fail/reallocate-bad-alignment-2.rs

```diff
@@ -1,16 +0,0 @@
-#![feature(alloc, allocator_api)]
```

Comment:
This is simply not possible with the new api

Reply:
How so? What's changed?

Reply:
`realloc` doesn't have a new alignment param anymore.

```rust
unsafe fn realloc(
    &self, 
    ptr: *mut Opaque, 
    layout: Layout, 
    new_size: usize
) -> *mut Opaque;
```